project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
```

Comment:
What's unclear here is if all docker projects can release patches whenever they want or if they (or some of them) have to depend on engine release. I'm thinking about boot2docker mainly here.